Makefile

```
@@ -24,6 +24,14 @@ lint:
 	#go tool vet .
 	golint $(pkgs) | grep -v 'unexported' || true
 
+fmt:
+	$(info INFO: Starting build $@)
+	{ \
+set -e ;\
+fmt=$$(gofmt -l .) ;\
+[ -z "$$fmt" ] && echo "valid gofmt" || (echo "invalid gofmt"; exit 1)\
```

```suggestion
[ -z "$$fmt" ] && echo "valid gofmt" || (echo -e "invalid gofmt\n$$fmt"; exit 1)\
```

When I ran it it just said: "invalid gofmt" without any other details, this adds the actual errors.

---

Great catch!